### DISCUSSION TITLE
Welcome to 🛡️ Armoria discussions!

### DISCUSSION THREAD

**Discussion by Azgaar** [Azgaar/Armoria#15] [General]
## 👋 Welcome!
  Let's have a single page for project-related discussions. It will be useful for ideas collection, Q&A and to build up a community. So please just comment below or start a new discussion to
  * Ask questions you’re wondering about
  * Share ideas
  * Tell about yourself and how you are using the Tool
  * Engage with community members.

Please note this is an _additional_ discussion platform to our usual [Discord](https://discord.com/invite/X7E84HU) and [Reddit](https://www.reddit.com/r/FantasyMapGenerator).

**Comment by Azgaar** [comment]
Here is a link we are using to collect new charges suggestions:
https://docs.google.com/spreadsheets/d/1jhwhRnut3ubPQ9cO3ph1wDwLPvWC607U_LjHA8_-YMg/edit#gid=0

**Comment by Avengium** [comment]
I like Armoria. I will continue suggesting charges on the spreadsheet and maybe i will try to make an svg myself.
